app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -296,7 +304,8 @@ public function __construct(
         \Magento\Sales\Api\OrderManagementInterface $orderManagement,
         \Magento\Quote\Model\QuoteFactory $quoteFactory,
         array $data = [],
-        \Magento\Framework\Serialize\Serializer\Json $serializer = null
+        \Magento\Framework\Serialize\Serializer\Json $serializer = null,
+        ExtensibleDataObjectConverter $dataObjectConverter = null
```

Comment:
Please, update docblock accordingly with the parameters that you accept.

Reply:
Done.